GG/GG/Layout.h

```diff
@@ -242,6 +242,10 @@ class GG_API Layout : public Wnd
     virtual void MouseWheel(const Pt& pt, int move, Flags<ModKey> mod_keys);
     virtual void KeyPress(Key key, boost::uint32_t key_code_point, Flags<ModKey> mod_keys);
     virtual void KeyRelease(Key key, boost::uint32_t key_code_point, Flags<ModKey> mod_keys);
+
+    /** Do all the layout work*/
+    virtual void DoLayout(Pt ul, Pt lr);
```

Comment:
if you're going to comment, explain what doing layout means; don't just rephrase the function name...